Alert/doc/spec.md

```diff
@@ -0,0 +1,148 @@
+| When a car suddenly stops | | | | | |
+
+In order to represent at data model the subtype of alert, the attribute "eventObserved" have been created. In this attribute, the information of subtype of alert should be added. It is necessary to define a catalogue of alerts and sub-types of alerts.
+
+## Data model
+
+A JSON Schema corresponding to this data model can be found [JSON Schema](../schema.json)
+
++ `id` : Entity's unique identifier.
+
++ `type` : Entity type. It must be equal to `Alert`.
+
++ `alertType` : Define the type of alert (Traffic jam, accidents, weather conditions, high level of pollutants)
+	+ Attribute type: text
+    + Normative References: [https://schema.org/Event](https://schema.org/Event)
+    + Mandatory
+
++ `eventObserved` : Describe the subtype of alerts (Alert: Traffic jam, Subtypes of alerts: Standstill Traffic Jam, Heavy Traffic Jam,  Car Stopped On Road, and so on).
+	 + Attribute type: text
+	 + Normative References: [https://schema.org/subEvent](https://schema.org/subEvent)
+	 + Mandatory
+
++ `location` : Location of alert represented by a GeoJSON geometry.
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory
+
++ `address` : Civic address of `Alert`
+    + Normative References: [https://schema.org/address](https://schema.org/address)
+    + Mandatory if `location` is not present.
+
++ `dateTime` : The date and time of this alert in ISO8601 UTCformat. It can be represented by an specific time instant or by an ISO8601 interval.
+	+ Attribute type: DateTime or an ISO8601 interval represented as Text.
+	+ Mandatory
+
++ `description` : A description of alert (Traffic jam in Paseo de la Reforma. Emergency services at place).
+	 + Attribute type: text
+	 + Normative References: [https://schema.org/description](https://schema.org/description)
+	 + Optional	 
+
++ `refUser` : reference to the User that generated an alert. Within FIWARE, this reference could point to the end of the FIWARE Identity Manager, where FIWARE user profiles are stored.
+    + Attribute type: `string`
+    + Mandatory	if `refDevice` is not present.
+
++ `refDevice` : reference to device that generate an alert.
+    + Attribute type: `string`
+    + Mandatory	if `refUser` is not present.
+
++ `data` : used to carry additional data for the alert.
+    + Attribute type: `StructuredValue`
+    + optional.
+
++ `severity` : a catalogue created by the application owner to describe the scale of an Alert reported in severityIndex expressed as a Text array . (Ex. Severity: 0:Informational, 1:Low, 2:Medium, 3:High, 4:Critical).
+    + Attribute type: `Text`
+    + optional. Mandatory if `severityIndex` is present.
+
++ `severityIndex` : number used to describe the severity of an Alert . (Ex. severityIndex: integer in range 0-4).
+    + Attribute type: `Number`
+    + optional. Mandatory if `severity` is present.
+
```

Comment:
I don't know if we need two different attributes for stating the same thing, if not I would drop one of them